assets/js/popover.js

```diff
@@ -0,0 +1,34 @@
+function htmlToElement(html) {
+    const template = document.createElement('template')
+    html = html.trim()
+    template.innerHTML = html
+    return template.content.firstChild
+}
+
+function initPopover(base) {
+    const baseUrl = base.replace(window.location.origin, "") // is this useless?
```

Comment:
no this helps to fix cases for when Quartz is deployed on a path like`/quartz`

Reply:
Oh, I didn't think about that. Commit 978d5ca removes the comment and renames the variables to make the intent a little clearer.